code/modules/ruins/lavalandruin_code/hermit.dm

```diff
@@ -25,27 +25,34 @@
 			only one pod left when you got to the escape bay. You took it and launched it alone, and the crowd of terrified faces crowding at the airlock door as your pod's engines burst to \
 			life and sent you to this hell are forever branded into your memory."
 			outfit.uniform = /obj/item/clothing/under/misc/assistantformal
+			outfit.suit = /obj/item/clothing/suit/space/syndicate/orange
+			outfit.head = /obj/item/clothing/head/helmet/space/syndicate/orange
```

Comment:
instead of duplicating this for all of these, take out the common items like suit, head, and back, and pit them after or before the switch statement